src/generateHelpUrl.ts

```diff
@@ -1,5 +1,5 @@
 const BASE_URL = 'https://docs.sanity.io/help/'
```

Comment:
```suggestion
const BASE_URL = 'https://www.sanity.io/help/'
```

The `docs.sanity.io` subdomain now just redirects to `sanity.io/docs`. The new format is `sanity.io/help`. For example: [https://www.sanity.io/help/studio-v2-vs-v3](https://www.sanity.io/help/studio-v2-vs-v3).